skops/io/_sklearn.py

```diff
@@ -97,17 +103,18 @@ def __init__(self, state, load_context: LoadContext, constructor, trusted=False)
             "constructor": constructor,
         }
 
-    def _construct(self):
+    def _construct(self) -> Any:
         args = self.children["args"].construct()
-        instance = self.children["constructor"](*args)
+        constructor = self.children["constructor"]
+        instance = constructor(*args)
```

Comment:
because it's more readable?

Reply:
I wanted to use the `constructor` variable below in the raised error, otherwise the line would have been too long, I would have to split it and make it less readable.